### DISCUSSION TITLE
现在 giscus 还会自动创建 Discussions 吗？

### DISCUSSION THREAD

**Discussion by llnancy** [vuepress-theme-hope/vuepress-theme-hope#3261] [Q&A]
[https://plugin-comment2-demo.vuejs.press/test2.html](https://plugin-comment2-demo.vuejs.press/test2.html)

在演示站的test2页面评论，并不会出现在页面上

<img width="1435" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/40930677/4cb85904-03f1-438d-8846-8cf0b67706f7">

**Comment by Mister-Hope** [comment]
你确定你完成了评论？
